CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Fix specific product voucher in draft orders - #5727 by @fowczarek
 - Explicit country assignment in default shipping zones - #5736 by @maarcingebala
 - Drop `json_content` field from the `Menu` model - #5761 by @maarcingebala
-- Refactor JWT support - #5734 by @korycins
+- Refactor JWT support - These changes could require a handling JWT token in the storefront. Storefront needs to handle a case when the backend returns the exception about the invalid token. - #5734, #5816 by @korycins
```

Comment:
We we mention this in the changelog? There are usually some changes in the backend that require changes in frontend as well.

Reply:
I thought that this will be a good idea to inform the community that some changes from the changelog will require extra actions.  A similar case was with the Vatlayer, where we deactivate the plugin but we didn't mention it explicitly. 
If you don't like this, then I will revert it.

Reply:
It's a good idea, I'm just thinking what is the best place to mention this. Here it may be hard to find, but maybe we should extract these changes required for the frontend when releasing a new version and add some section "Frontend breaking changes" or something.

Reply:
Or just a large "breaking changes" section at the very top of the changelog? Once Cloud becomes public we will have to withhold any breaking changes until the next major release.